tutorials/run-laravel-on-appengine-standard/index.md

```diff
@@ -0,0 +1,259 @@
+
+        gcloud sql instances describe YOUR_INSTANCE_NAME | grep connectionName
+
+    * Start the Cloud SQL proxy and replace `YOUR_CONNECTION_NAME` with the connection name you retrieved in the previous step.
+
+        cloud_sql_proxy -instances=YOUR_CONNECTION_NAME=tcp:3306
+
+    * Use `gcloud` to create a database for the application.
+
+        gcloud sql databases create laravel --instance=YOUR_INSTANCE_NAME
+
+1. Run the database migrations for Laravel. This can be done locally by setting
+  your parameters in `.env` or by passing them in as environment variables. Be
+  sure to replace `YOUR_DB_PASSWORD` below with the root password you
+  configured:
+
+    ```sh
+    # create a migration for the session table
+    php artisan session:table
+    export DB_DATABASE=laravel DB_USERNAME=root DB_PASSWORD=YOUR_DB_PASSWORD
+    php artisan migrate --force
+    ```
+
+1. Modify your `app.yaml` file with [the following contents][app-dbsessions-yaml]:
+C
+1. Replace each instance of `YOUR_DB_PASSWORD` and `YOUR_CONNECTION_NAME`
+   with the values you created for your Cloud SQL instance above.
+
+## Set up Stackdriver Logging and Error Reporting
+
+Before we begin, install both of the Google Cloud client libraries for Stackdriver
+Logging and Error Reporting:
+
+    composer require google/cloud-logging google/cloud-error-reporting
+
+### Stackdriver Logging
+
+You can write logs to Stackdriver Logging from PHP applications by using the Stackdriver Logging library for PHP directly.
+
+1. First, create a custom logger in
+  [`app/Logging/CreateStackdriverLogger.php`][app-logging-createstackdriverlogger-php]:
+
+    ```php
+    namespace App\Logging;
+
+    use Google\Cloud\Logging\LoggingClient;
+    use Monolog\Handler\PsrHandler;
+    use Monolog\Logger;
+
+    class CreateStackdriverLogger
+    {
+        /**
+         * Create a custom Monolog instance.
+         *
+         * @param  array  $config
+         * @return \Monolog\Logger
+         */
+        public function __invoke(array $config)
+        {
+            $logName = isset($config['logName']) ? $config['logName'] : 'app';
+            $psrLogger = LoggingClient::psrBatchLogger($logName);
+            $handler = new PsrHandler($psrLogger);
+            $logger = new Logger($logName, [$handler]);
+            return $logger;
+        }
+    }
+    ```
+
+1. Next, you'll need to add our new custom logger to the `channels` array in
+  [`config/logging.php`][config-logging-php]:
+
+    ```php
+    'channels' => [
+
+        // Add the following lines to integrate with Stackdriver:
+        'stackdriver' => [
+            'driver' => 'custom',
+            'via' => App\Logging\CreateCustomLogger::class,
+            'level' => 'debug',
+        ],
+    ```
+
+1. Now you can log to Stackdriver logging anywhere in your application!
```

Comment:
It's not a big deal, but this isn't consistent with your capitalization elsewhere of 'Stackdriver Logging'.